src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/scriptrunner/scriptrunner.py

```diff
@@ -26,27 +28,31 @@ def _main(settings, optlist, args):
             xunitrunner
                 [--config config.json]
                 [--setting name=value]
-                --script
-                [--args arg1 arg2...]
+                --script=path
+                [--standalone]
```

Comment:
>standalone [](start = 19, length = 10)

How does the repro tool know to all --standalone?

Reply:
Maybe I should have included in the original description.. there is a parallel PR in the CoreFX Infra repo: https://mseng.visualstudio.com/Tools/Engineering%20Infrastructure/_git/CoreFX%20Engineering%20Infrastructure/pullrequest/285825?_a=overview